extension/src/path/selection/tree.ts

```diff
@@ -90,6 +92,9 @@ export abstract class BasePathSelectionTree<
     if (description) {
       treeItem.description = description
     }
+    if (tooltip) {
+      treeItem.tooltip = tooltip
```

Comment:
What's the point of checking for the element before assigning it?  If `tooltip` is `undefined`, `treeItem.tooltip` is `undefined` as well anyway, no? I'm asking because I'm probably missing something.

Reply:
you are right, there is no point. I'll update